pkgs/applications/networking/instant-messengers/telegram/kotatogram-desktop/default.nix

```diff
@@ -58,6 +58,7 @@ stdenv.mkDerivation rec {
   patches = [
     ./macos.patch
     ./macos-opengl.patch
+    ./macos-qt5.patch
```

Comment:
Are there upstream commits/pr for this?

Reply:
No. Nor I guess any of upstreams really need it as they build macOS builds with Qt 6.2.